packages/template-retail-react-app/app/components/experience/image-with-text/index.jsx

```diff
@@ -0,0 +1,132 @@
+                            src={image?.src?.mobile ? image?.src?.mobile : image?.url}
+                            ignoreFallback={true}
+                            alt={alt}
+                            title={alt}
+                            filter={'brightness(40%)'}
+                        />
+                    </Link>
+                </picture>
+                {(ITCText || heading) && (
+                    <Text as="figcaption">
+                        {heading && (
+                            <Box
+                                className={'image-with-text-heading-container'}
+                                position={'absolute'}
+                                top={'50%'}
+                                width={'100%'}
+                            >
+                                <Text
+                                    as="span"
+                                    className={'image-with-text-heading-text'}
+                                    color={'white'}
+                                >
+                                    <Box
+                                        dangerouslySetInnerHTML={{
+                                            __html: sanitizedHtml?.heading
+                                        }}
+                                        sx={{
+                                            p: {
+                                                display: 'flex',
+                                                alignItems: 'center',
+                                                justifyContent: 'center'
+                                            }
+                                        }}
+                                    />
+                                </Text>
+                            </Box>
+                        )}
+                        {ITCText && (
+                            <Box>
+                                <Text as="span" className={'image-with-text-text-underneath'}>
+                                    <Box
+                                        dangerouslySetInnerHTML={{
+                                            __html: sanitizedHtml?.ITCText
+                                        }}
+                                        sx={{
+                                            p: {
+                                                display: 'flex',
+                                                alignItems: 'center',
+                                                justifyContent: 'center'
+                                            }
+                                        }}
+                                    />
+                                </Text>
+                            </Box>
+                        )}
+                    </Text>
+                )}
+            </Box>
+        </div>
+    )
+}
+
+ImageWithText.propTypes = {
+    ITCLink: PropTypes.string,
+    ITCText: PropTypes.string,
+    image: PropTypes.shape({
```

Comment:
I believe this is a required prop. Can you verify this and the other props so we have them all set up correctly.